target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vrv9510kwac23.dtsi

```diff
@@ -0,0 +1,284 @@
+#include "vr9.dtsi"
```

Comment:
The SPDX license identifier is also missing here.

Reply:
Done